.gitignore

```diff
@@ -4,6 +4,7 @@ vendor/pkg
 contrib/cmd/recvtty/recvtty
 contrib/cmd/sd-helper/sd-helper
 contrib/cmd/seccompagent/seccompagent
+contrib/cmd/fs_idmap/fs_idmap
```

Comment:
nit: can you please use `-` not `_`, just for the uniformity sake (see `sd-helper` above)?

Reply:
This was solved already :)